src/components/BarChart.js

```diff
@@ -10,8 +10,11 @@ const Bar = React.createClass({
   propTypes: {
     animateOnHover: React.PropTypes.bool,
     animationDuration: React.PropTypes.number,
+    hasNegative: React.PropTypes.bool,
+    hasPositive: React.PropTypes.bool,
```

Comment:
It's on the state for the BarChart, still need to pass the prop down to the Bar component.